src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/RootController.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collections;
+import java.util.Map;
+
+@RestController
+public class RootController {
+
+    /*
+     * Azure hits us on / every 5 seconds to prevent it sleeping the application
+     * Application insights registers that as a 404 and adds it as an exception,
+     * This is here to reduce the noise
+     */
+    @GetMapping(value = "/", consumes = MediaType.ALL_VALUE, produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public Map root() {
+        return Collections.singletonMap("hello", "world!");
```

Comment:
I would not expect to see this in production grade application. I would expect either blank page / empty JSON object or page / JSON object with some explanation why this path exists.

Reply:
@geek-soft removed